### PR TITLE
Update Laplace bridge predictive

### PR DESCRIPTION
**Pull request by runame**
Adjust LA bridge predictive to latest version of the paper. Thanks to @mariushobbhahn for the code blueprint.

I have also added a check to make sure people don't think they are using any other link approximation than `'mc'` together with `pred_type='nn'`.

### CODE REVIEW

**Review by AlexImmer — APPROVED** [review]
Looks good to me as far as I can judge it. Did you try it on one of the example scripts already?

File: laplace/baselaplace.py [hunk 1]
```diff
@@ -554,13 +557,24 @@ def __call__(self, x, pred_type='glm', link_approx='probit', n_samples=100,
...
-                return torch.nan_to_num(dist.mean, nan=1.0)
+                # optional: variance correction
+                if link_approx == 'bridge_norm':
+                    f_var_diag_mean = f_var_diag.mean(dim=1)
+                    f_var_diag_mean /= torch.as_tensor([K/2], device=self._device).sqrt()
+                    f_mu /= f_var_diag_mean.sqrt().unsqueeze(-1)
+                    f_var_diag /= f_var_diag_mean.unsqueeze(-1)
+                sum_exp = torch.exp(-f_mu).sum(dim=1).unsqueeze(-1)
+                alpha = (1 - 2/K + f_mu.exp() / K**2 * sum_exp) / f_var_diag
+                return torch.nan_to_num(alpha / alpha.sum(dim=1).unsqueeze(-1), nan=1.0)
```
[the remark this answers is not in the document]
Comment (runame) [reply laplace/baselaplace.py:577]: I guess it depends on the use case. Mabye a question for @mariushobbhahn: do you think this would be useful?

File: laplace/baselaplace.py [hunk 2]
```diff
@@ -510,7 +510,7 @@ def __call__(self, x, pred_type='glm', link_approx='probit', n_samples=100,
             network sampling predictive. The GLM predictive is consistent with
             the curvature approximations used here.
 
-        link_approx : {'mc', 'probit', 'bridge'}
+        link_approx : {'mc', 'probit', 'bridge', 'bridge_norm'}
```
[the remark this answers is not in the document]
Comment (mariushobbhahn) [reply laplace/baselaplace.py:513]: Runa is correct. "norm" stands for "normalized" and refers to a correction factor we introduce in the latest version of the paper. Will be on arxiv next week.

File: laplace/baselaplace.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariushobbhahn) [reply laplace/baselaplace.py:577]: The Laplace Bridge returns a Dirichlet, so technically, we could provide the alpha-vector and could calculate the mean, mode, variance, entropy, etc. from that. However, most of the time, people just want to use the mean and you returned the mean in the previous version, so I kept that.
